RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -87,17 +86,6 @@ public CodeExplorerItemViewModel SelectedItem
                 _selectedItem = value;
                 OnPropertyChanged();
 
-                if (_selectedItem is CodeExplorerProjectViewModel)
-                {
-                    var vbe = _selectedItem.GetSelectedDeclaration().Project.VBE;
-                    var project = vbe.VBProjects.Cast<VBProject>().FirstOrDefault(f => f.HelpFile == _selectedItem.GetSelectedDeclaration().Project.HelpFile);
-
-                    if (project != null)
-                    {
-                        vbe.ActiveVBProject = project;
-                    }
-                }
-
```

Comment:
Actually, the VBE somehow (or it did, anyway) update the selected project when a node was right-clicked.  It won't be hard to set the selected project in the execute for that command, anyway.